recipes/euphonic/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "euphonic" %}
+{% set version = "0.5.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/euphonic-{{ version }}.tar.gz
+  sha256: f6cded258acdc6beec318a5dc3f076796cb1d5ba6a1654bac93c3ce704248612
+
+build:
+  number: 0
+  entry_points:
+    - euphonic-dispersion = euphonic.cli.dispersion:main
+    - euphonic-dos = euphonic.cli.dos:main
+    - euphonic-optimise-eta = euphonic.cli.optimise_eta:main
+    - euphonic-show-sampling = euphonic.cli.show_sampling:main
+    - euphonic-intensity-map = euphonic.cli.intensity_map:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - numpy >=1.12.1
+    - pip
+    - python
+  run:
+    - importlib_resources >=1.3.0
+    - {{ pin_compatible('numpy') }}
+    - pint >=0.9
+    - python
+    - scipy >=1.0.0
+    - seekpath >=1.1.0
+    - threadpoolctl >=1.0.0
+
+test:
+  imports:
+    - euphonic
+    - euphonic.cli
+    - euphonic._euphonic
+  commands:
+    - pip check
+    - euphonic-dispersion --help
+    - euphonic-dos --help
+    - euphonic-optimise-eta --help
+    - euphonic-show-sampling --help
+    - euphonic-intensity-map --help
+  requires:
+    - pip
+    - matplotlib >=2.0.0
```

Comment:
Why is this only required here? Also would `matplotlib-base` suffice (probably)?

Reply:
matplotlib is an optional dependency for this package; if people only want to use it for fast interpolation of phonon data then it's a rather heavy dependency to drag in. It is however required for the command-line tools (which present plots), and therefore is needed for the tests of those tools.

Reply:
Matplotlib-base should be fine, good shout.